src/pages/Register.tsx

```diff
@@ -1,42 +1,51 @@
 import { useState, FormEvent } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import { UserPlus } from 'lucide-react';
 import Button from '../components/Button';
 import Input from '../components/Input';
 import toast from 'react-hot-toast';
-import axios from 'axios'
+import { authService } from '../services/api';
 
 export default function Register() {
+  const navigate = useNavigate();
   const [isLoading, setIsLoading] = useState(false);
-  const [firstName, setfirstName] = useState("")
-  
+  const [formData, setFormData] = useState({
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    password: '',
+    confirmPassword: ''
+  });
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setFormData(prev => ({
+      ...prev,
+      [name]: value
+    }));
+  };
 
   const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     setIsLoading(true);
 
-
-    // /register
-    // const responce = fetch()
-    // 
-    const data = {
-      firstName
-
-    }
-    const config = {
-      
+    if (formData.password !== formData.confirmPassword) {
+      toast.error('Passwords do not match!');
+      setIsLoading(false);
+      return;
     }
-    axios.post(`https://medcaresystem-5sag--5173--d3acb9e1.local-credentialless.webcontainer.io/register`, data, config)
-    .then(response => {
-      console.log('Response:', response.data);
-    })
-    .catch(error => {
-      console.error('Error:', error);
-    });
-
 
-
-    setIsLoading(false);
+    try {
+      const { confirmPassword, ...registerData } = formData;
```

Comment:
Once register the user is re-routed or sent to the  Login page for continuing the Login process